### ISSUE TITLE
Unable to delete a schedule in scheduler card

### ISSUE THREAD

**Issue by Strangehadron** [nielsfaber/scheduler-card#561]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

2.3.4

### Component Version

3.2.9

### Bug description

Hi, i'm unable to delete a schedule in my scheduler card with the delete option. It give me an error in a popup and the schedule isn't delete.

> Something went wrong!
> Response error: 500
> Please [report](https://github.com/nielsfaber/scheduler-card/issues) the bug

Tried to clean my cache and reinstall card and scheduler component with no success.

### Steps to reproduce

- Pick a schedule in scheduler card
- Click delete

### Additional info

Home Assistant Core 2022.6.0
Home Assistant Supervisor 2022.05.3
Home Assistant OS 8.1

Also got this in my log but don't know if related because not log at the same moment :

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:103 
First occurred: 04:56:10 (2 occurrences) 
Last logged: 05:22:59
Detected integration that uses deprecated `async_get_registry` to access device registry, use async_get instead. Please report issue to the custom component author for alarmo using this method at custom_components/alarmo/__init__.py, line 58: device_registry = await dr.async_get_registry(hass)
Detected integration that uses deprecated `async_get_registry` to access device registry, use async_get instead. Please report issue to the custom component author for scheduler using this method at custom_components/scheduler/__init__.py, line 47: device_registry = await dr.async_get_registry(hass)
```
EDIT : 
I have not see that one first : 

```
Logger: aiohttp.server
Source: custom_components/scheduler/__init__.py:275 
Integration: Scheduler integration ([documentation](https://github.com/nielsfaber/scheduler-component), [issues](https://github.com/nielsfaber/scheduler-component/issues)) 
First occurred: 05:27:13 (7 occurrences) 
Last logged: 05:42:01

Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 100, in forwarded_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 79, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 220, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 137, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 62, in wrapper
    result = await method(view, request, *args, **kwargs)
  File "/config/custom_components/scheduler/websockets.py", line 64, in post
    await coordinator.async_delete_schedule(data[const.ATTR_SCHEDULE_ID])
  File "/config/custom_components/scheduler/__init__.py", line 275, in async_delete_schedule
    entity_registry = await get_entity_registry(self.hass)
TypeError: object EntityRegistry can't be used in 'await' expression
```
Also in the integration page, my entities are "control" type, used to be "config" last time i've checked.

Thanks for your help ! (sorry if this one needed to be post in scheduler component repo)

**Comment by zuchie-star** [comment]
I also had this issue yesterday when i tried to delete.

**Comment by nielsfaber** [comment]
This should be fixed by updating the scheduler-component to v3.2.10
